Firmware/ControladorRobo/ControladorRobo.ino

```diff
@@ -64,6 +64,7 @@ void setup()
         while (millis()<calibrationTimer)
         {
             infrared->startCalibration();
+            infrared->readSensors();
         }
         infrared->endCalibration();
```

Comment:
I am not saying the we need to this now, but we need a top level like `robot` the it will deal with this high-level functions like calibration,